libraries/abstractions/common_io/include/iot_uart.h

```diff
@@ -159,7 +160,7 @@ typedef struct
  *  IotUARTHandle_t xOpen;
  *  int32_t lRead, lWrite, lClose;
  *  BaseType_t xCallbackReturn;
- *  uint8_t ucPort = 1; /* Each UART peripheral will be assigned an integer.
+ *  uint8_t ucPort = 1; // Each UART peripheral will be assigned an integer.
```

Comment:
Nit: looks like a small typo, can this be reverted

Reply:
I intentionally changed it to //, because it won't compile for our iot/vendor/dev with /*

Reply:
Does AFR compile it successfully?

Reply:
Oh I see, it was missing the ending */. Looks good to me

Reply:
Unfortunately AFR doesn't compile common io headers for now.